frontend/src/pages/home.ts

```diff
@@ -249,23 +247,29 @@ export class Home extends LiteElement {
   }
 
   private renderInvite() {
-    if (this.isInviteComplete) {
-      return html`
-        <sl-button @click=${() => (this.isInviteComplete = false)}
-          >${msg("Send another invite")}</sl-button
-        >
-      `;
-    }
-
-    const defaultOrg = this.userInfo?.orgs.find(
-      (org) => org.default === true,
-    ) || { name: "" };
     return html`
       <btrix-invite-form
         .authState=${this.authState}
         .orgs=${this.orgList}
-        .defaultOrg=${defaultOrg}
-        @success=${() => (this.isInviteComplete = true)}
+        @btrix-invite-success=${(e: CustomEvent<InviteSuccessDetail>) => {
+          const org = this.orgList?.find(({ id }) => id === e.detail.orgId);
+
+          this.notify({
+            message: msg(
+              html`Invite sent!
+                <br />
+                <a
+                  class="underline hover:no-underline"
+                  href="/orgs/${org?.slug || e.detail.orgId}/settings/members"
+                  @click=${this.navLink.bind(this)}
+                >
+                  View org members
+                </a> `,
+            ),
```

Comment:
I think we'd make translators' lives easier if we just wrapped the text in `msg`s, rather than the entire block here